run_scans.py

```diff
@@ -21,7 +22,7 @@ def launchTestsArgs(tool, shelf, slot, link, chamber, scanmin, scanmax, nevts,
   #Build Commands
   setupCmds = []
   preCmd = None
-  cmd = ["%s"%(tool),"-s%d"%(slot),"-g%d"%(link),"--shelf=%i"%(shelf)]
+  cmd = ["%s"%(tool),"-s%d"%(slot),"-g%d"%(link),"--shelf=%i"%(shelf), "--nevts=%i"%(nevts), "--vfatmask=%s"%(vfatmask)]
```

Comment:
you've tested this?  some places you're passing it as an `int` (`%i`), some places as a `string` (`%s`)
I'd have probably naively gone with `hex` (`%x` or `0x%x`)